### PR TITLE
Redesignating 1.0.0.11 to be a Minor Bugfix Update

### PR DESCRIPTION
**Pull request by RavenholmZombie**
baba booey

### CODE REVIEW

File: src/main/java/com/mesabrook/ib/init/XNetAPI.java [hunk 1]
```diff
@@ -44,4 +45,17 @@ public ConnectResult canConnect(IBlockAccess world, BlockPos connectorPos, Block
...
 	}
+	
+	public static class SoundEmitterConnectable implements IConnectable
+	{
+		@Override
+		public ConnectResult canConnect(IBlockAccess world, BlockPos connectorPos, BlockPos blockPos, TileEntity te, EnumFacing facing)
+		{
+			if (te instanceof TileEntitySoundEmitter)
+			{
+				return (facing == EnumFacing.UP || facing == EnumFacing.DOWN || facing == EnumFacing.NORTH || facing == EnumFacing.SOUTH || facing == EnumFacing.EAST || facing == EnumFacing.WEST) ? ConnectResult.YES : ConnectResult.NO;
```
[the remark this answers is not in the document]
Comment (RavenholmZombie) [reply src/main/java/com/mesabrook/ib/init/XNetAPI.java]: Fixed in https://github.com/Mesabrook-Development-Team/Immersibrook/pull/76/commits/63daa9e55d369f374307693dca5ce7ad5026dd60

File: src/main/java/com/mesabrook/ib/init/XNetAPI.java [hunk 2]
```diff
@@ -44,4 +45,17 @@ public ConnectResult canConnect(IBlockAccess world, BlockPos connectorPos, Block
 			return ConnectResult.DEFAULT;
 		}
 	}
+	
+	public static class SoundEmitterConnectable implements IConnectable
```
[the remark this answers is not in the document]
Comment (RavenholmZombie) [reply src/main/java/com/mesabrook/ib/init/XNetAPI.java:50]: Fixed in https://github.com/Mesabrook-Development-Team/Immersibrook/pull/76/commits/63daa9e55d369f374307693dca5ce7ad5026dd60

File: src/main/java/com/mesabrook/ib/blocks/ImmersiblockRotational.java [hunk 3]
```diff
@@ -209,11 +210,11 @@ public void addInformation(ItemStack stack, @Nullable World world, List<String>
             {
                 tooltip.add(TextFormatting.YELLOW + "The brain of the Self-Checkout system.");
             }
-            if(this == ModBlocks.SCO_SCANNER)
+            if(this.getUnlocalizedName().contains("scanner"))
```
[the remark this answers is not in the document]
Comment (RavenholmZombie) [reply src/main/java/com/mesabrook/ib/blocks/ImmersiblockRotational.java]: Fixed in https://github.com/Mesabrook-Development-Team/Immersibrook/pull/76/commits/63daa9e55d369f374307693dca5ce7ad5026dd60

**Review by CSX8600 — APPROVED** [review]
bfd6d06cf6af95c96be25204948f9b7a 1

**Pull request merged**